expo/expo
# [doctor] prevent showing eas docs if not using eas

# Why

Solves [ENG-13827](https://linear.app/expo/issue/ENG-13827/doctor-check-links-to-eas-build-docs-even-for-projects-not-using-eas)

# How

Added a ternary check to validate if the project uses EAS; if so, it displays the relevant documentation.

# Test Plan

- Run doctor locally
- Prebuild the app
- Run the check and verify it doesn't show the link
- Init EAS project
- Re-run check and verify link shows

<img width="1034" alt="image" src="https://github.com/user-attachments/assets/cf859c31-0a53-4c7f-9717-b86c7fcee712">

# Checklist
